webapp/src/ts/components/search-bar/search-bar.component.ts

```diff
@@ -31,12 +41,17 @@ export class SearchBarComponent implements AfterContentInit, OnDestroy {
   constructor(
     private store: Store,
     private responsiveService: ResponsiveService,
+    private searchFiltersService: SearchFiltersService,
   ) { }
 
   ngAfterContentInit() {
     this.subscribeToStore();
   }
 
+  ngAfterViewInit() {
+    this.searchFiltersService.init(this.freetextFilter);
```

Comment:
The element is ready in the DOM in `ngAfterViewInit` and not before.
While we should subscribe to the store after the component is inited (`ngAfterContentInit`), this avoids the Angular's `expression has changed exception` that we resolved a while ago as part of another ticket.